docs/src/main/asciidoc/security-basic-authentication-tutorial.adoc

```diff
@@ -165,8 +168,8 @@ public class AdminResource {
     }
 }
 ----
-* Finally, implement the `/api/users/me` endpoint. As you can see from the source code example below, we are trusting only users with the `user` role.
-We are also using `SecurityContext` to get access to the currently authenticated `Principal`, and we return the username, all of which is loaded from the database.
+* Finally, implement an `/api/users/me` endpoint that trusts users who have the `user` role.
```

Comment:
```suggestion
* Finally, implement an `/api/users/me` endpoint that can only accessed by users who have the `user` role.
```

Reply:
It does not sound quite right,  the endpoints themselves are not involved in the RBAC process

Reply:
@sberyozkin - Fixed. Thanks for your suggestion.